src/kt_class.h

```diff
@@ -43,8 +43,10 @@ class KtClass : public JavaInstanceWrapper<KtClass> {
 
     bool is_assignable_from(KtClass* p_class) const;
 
+    const Dictionary get_rpc_config();
 private:
     HashMap<StringName, KtFunction*> methods;
+    HashMap<StringName, RpcConfig> rpc_method_configs;
```

Comment:
Do we really need that HashMap when we already have the Rpc configs set in the Kfunction instances ? We have a duplicate of data here.

Reply:
True. Will remove the HashMap. Makes it simpler anyways.